cypress/utils/plugins/dashboards-assistant/commands.js

```diff
@@ -135,7 +135,10 @@ Cypress.Commands.add('registerRootAgent', () => {
 });
 
 Cypress.Commands.add('putRootAgentId', (agentId) => {
-  if (Cypress.env('SECURITY_ENABLED')) {
+  if (
+    Cypress.env('SECURITY_ENABLED') &&
+    !Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')
```

Comment:
Can you elaborate more why we add this line?

Reply:
Sure, the annotation has been added.

Reply:
Thanks, I see, that makes sense.